### PR TITLE
Removes the ban on line splitting, adds a note about its proper use

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

I've removed the ban on line splitting, and added a note warning against mass conversions and such.

## Why It's Good For The ~~Game~~ Codebase

We still use it, and the tool has its place. If someone were to go through and replace all of our multi-line code the readability of our codebase would go down not up.

This isn't to say it should be used on all long lines, or that we should have a line limit. We should just leave the option open.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -556,6 +556,8 @@ for (var/i in reagents)
 
 * Do not divide when you can easily convert it to multiplication. (ie `4/2` should be done as `4*0.5`)
 
+* The use of `\` to separate single lines into more readable blocks is not banned, however you should use it only where it makes new information more accessible. We do not have a column limit, and mass conversions will not be received well.
```
Comment (Mothblocks) [inline .github/CONTRIBUTING.md:559]: I'm 99% sure backslashes are only necessary for like, macros. I split lines normally a lot without them and they work fine. This should be a generic comment about multilining.

**Comment by Mothblocks** [comment]
@tgstation/commit-access

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (LemonInTheDark) [reply .github/CONTRIBUTING.md:559]: ~~It's needed for macros and math expressions, which are the main place I've seen them~~
Sure, you're right, this should be more general

**Comment by ZeWaka** [comment]
Big fan of this.

Looking forward to creating a mass-regex PR to convert every single line of code into a multiline.

**Comment by LemonInTheDark** [comment]
I will have your head

**Comment by Iamgoofball** [comment]
:-1:

line splitting should only be used for macros and for really long shell() commands

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Rohesie) [reply .github/CONTRIBUTING.md:559]: This is bad (assuming the variables are bigger than just single letters, of course, creating big horizontal drift):
```dm
var/list/codes = list(alpha, bravo, charlie, delta, echo, foxtrot)
```
this is better:
```dm
var/list/codes = list(
	alpha,
	bravo,
	charlie,
	echo,
	foxtrot,
)
```
It's quite clear and reduces chances of merge conflicts.

Comment (LemonInTheDark) [reply .github/CONTRIBUTING.md:559]: @Rohesie I'm not really sure what you're getting at here, do you want me to insert that as an example?

Comment (Mothblocks) [reply .github/CONTRIBUTING.md]: The end parenthese should lower an indentation level, for the record 😄

**Pull request merged**